lib/std/Target.zig

```diff
@@ -144,10 +144,29 @@ pub const Os = struct {
             };
         }
 
+        pub inline fn isMinGW(tag: Os.Tag, abi: Abi) bool {
+            return tag == .windows and abi.isGnu();
+        }
```

Comment:
don't think these added functions are justified

Reply:
I only added these for consistency. I would actually prefer to just delete these functions and only have the top-level ones on `std.Target`. Would you be ok with that?

Reply:
yes